Misc/NEWS.d/next/Library/2018-06-10-15-14-17.bpo-33805.5LAz5a.rst

```diff
@@ -0,0 +1 @@
+Update error message of dataclasses.replace() when InitVar is not specified.
```

Comment:
I'd prefer "Improve error message of dataclasses.replace() when an InitVar is not specified".